src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -353,6 +353,7 @@ public void SendFunctionEnvironmentReloadRequest()
 
         internal void SendFunctionLoadRequest(FunctionMetadata metadata)
         {
+            _functionInputBuffers[metadata.FunctionId] = new BufferBlock<ScriptInvocationContext>();
```

Comment:
Note: This is what we had before the multiple language workers refactor.

Reply:
Okay, so invocations will get buffered, then when they function is loaded and the dataflow is wired up fully, they'll be invoked?

Reply:
resolving from offline conversation. Yes. Invocations will be buffered until the action block is linked.